examples_test.go

```diff
@@ -2,7 +2,9 @@ package dotsql_test
 
 import (
 	"database/sql"
-	"github.com/gchaincl/dotsql"
+	"log"
+
+	dotsql "./" // You'd use "github.com/gchaincl/dotsql" in your code
```

Comment:
I'm not convinced about this, since local imports are discouraged.
What I normally do here is to git clone the repo and just add different remotes. In your case you should
```bash
cd $GOPATH/src
git clone github.com/gchaincl
git remote add ondrowan git@github.com:ondrowan/dotsql
```

and then just push to one remote or another